third_party/xla_client/stablehlo_helper.h

```diff
@@ -0,0 +1,12 @@
+#ifndef STABLEHLO_HELPER_H_
```

Comment:
Why do we add this to the xla_client instead of torch_xla?

Reply:
I found currently all TF dependencies used in PyTorch/XLA are wrapped in `libxla_computation_client.so`, and `_XLAC.so` links with `libxla_computation_client.so` for TF dependencies. The MLIR/StableHLO APIs can be considered TF dependencies now, so I put them in xla_client